unit-tests/src/main/scala/tests/NativeFramework.scala

```diff
@@ -1,8 +1,10 @@
 package tests
 
 import sbt.testing.{Fingerprint, Framework, Runner}
-import scala.scalanative.testinterface.PreloadedClassLoader
 
+import scala.scalanative.reflect.annotation.EnableReflectiveInstantiation
+
+@EnableReflectiveInstantiation
```

Comment:
This should not be necessary. `NativeFramework` extends `sbt.testing.Framework`, and the latter has `@EnableReflectiveInstantiation`. `NativeFramework` therefore receives this by inheritance.

Reply:
I thought so too, but for some reason this is not the case. Removing this annotation results in a InstantiationError at `FrameworkLoader`.

Following the `sbt.testing.Interface` import in this file gives two options in IntelliJ: the one from `test-interface-sbt-defs` of Scala Native and one from `org.scala-sbt:test-interface`. It seems that `unit-tests` project depends on both. Maybe it resolves to the second one, which does not have the annotation?

Reply:
Could you check `show unit-tests/test:dependencyClasspath`? Does it have an entry for the JVM version of sbt-testing-interface? If yes, the build needs to be fixed so that that doesn't happen.